apps/emqx/src/emqx_router_helper.erl

```diff
@@ -51,7 +51,7 @@
     cleanup_routes/1
 ]).
 
--record(routing_node, {name, const = unused}).
```

Comment:
this is also intended.

Reply:
Added the `-hank` attribute to ignore it.